vortex-array/src/array/sparse/mod.rs

```diff
@@ -179,7 +179,18 @@ impl AcceptArrayVisitor for SparseArray {
     }
 }
 
-impl ArrayStatisticsCompute for SparseArray {}
+impl ArrayStatisticsCompute for SparseArray {
+    fn compute_statistics(&self, stat: Stat) -> VortexResult<StatsSet> {
+        let mut values_stats = self.values().with_dyn(|a| a.compute_statistics(stat))?;
+        let fill_stats = if self.fill_value().is_null() {
+            StatsSet::nulls(self.len(), self.dtype())
+        } else {
+            ConstantArray::new(self.fill_scalar(), self.len()).compute_statistics(stat)?
+        };
+        values_stats.merge(&fill_stats);
```

Comment:
One minor not, merge is more like concatenate, in particular it uses order of arguments to determine min/max/sortedness. I have a feeling there's a subtle bug here unless fill is all at the end. We should rename that method to `merge_ordered` and do some custom logic here since the ordering of fill values matters